code/obj/flock/furniture.dm

```diff
@@ -177,18 +177,20 @@
 		if (istype(W, /obj/item/satchel) && length(W.contents))
 			..()
 		else if (!issilicon(user))
-			if (istype(user, /mob/living/critter/flock/drone))
-				if (W)
-					user.u_equip(W)
-					W.set_loc(src.loc)
-				else
-					return ..()
-			else if(user.drop_item())
+			if(user.drop_item())
```

Comment:
Isn't there some fuckery with drop_item still needing u_unequip or something? I remember needing to do it anyway and then pali being really confused as to why

Reply:
Idk, I just changed this because leah made an upstream pr to fix critters not being able to put items in storage containers, which seemed to make this not necessary

Reply:
since it's now identical to the parent implementation it should be fine